sdktests/server_side_persistence_base.go

```diff
@@ -6,13 +6,24 @@ import (
 	"github.com/redis/go-redis/v9"
 	"github.com/stretchr/testify/require"
 
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	consul "github.com/hashicorp/consul/api"
+	o "github.com/launchdarkly/sdk-test-harness/v2/framework/opt"
+
 	"github.com/launchdarkly/go-sdk-common/v3/ldcontext"
 	"github.com/launchdarkly/go-sdk-common/v3/ldvalue"
 	"github.com/launchdarkly/go-server-sdk-evaluation/v3/ldbuilders"
 	"github.com/launchdarkly/sdk-test-harness/v2/framework/ldtest"
 	"github.com/launchdarkly/sdk-test-harness/v2/servicedef"
 )
 
+const (
+	PersistenceInitedKey = "$inited"
```

Comment:
This is now a public constant accessible from other packages (capital `P`). Unclear if this is intended?